sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/models/VectorStoreChunkingStrategyRequest.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.assistants.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * An abstract representation of a vector store chunking strategy configuration.
+ */
+@Immutable
+public class VectorStoreChunkingStrategyRequest implements JsonSerializable<VectorStoreChunkingStrategyRequest> {
```

Comment:
Do we need this class? It is just a wrapper for the enum type.

Reply:
Same as above, this type is polymorphic and when the discriminator `type` is `static` then we have an additional object to deserialize.